source/code/plugins/filter_security_baseline.rb

```diff
@@ -0,0 +1,40 @@
+
+require_relative 'security_baseline_lib'
+require_relative 'oms_common' 
+
+module Fluent
+  class SecurityBaselineFilter < Filter
+
+    Fluent::Plugin.register_filter('filter_security_baseline', self)
+
+    def configure(conf)
+        super
+        # Do the usual configuration here
+        @hostname = OMS::Common.get_hostname or "Unknown host"
+    end
+
+    def start
+        super
+        # This is the first method to be called when it starts running
+        # Use it to allocate resources, etc.
+        OMS::SecurityBaseline.log = @log
+    end
+
+    def shutdown
+        super
+        # This method is called when Fluentd is shutting down.
+        # Use it to free up resources, etc.
+    end
+
+    def filter(tag, time, record)       
+        # Create Security Baseline and Security Baseline Summary blobs based on omsbaseline tool scan & assessment results
+        security_baseline_blob, security_baseline_summary_blob = OMS::SecurityBaseline.transform_and_wrap(record, @hostname, time)
```

Comment:
The best practice is not to use static class or static methods in filter. It will block the reuse of the filter if you need to save any state.